samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -157,34 +157,63 @@ struct metric {
     } end;
 };
 
+#define NUM_SAMPLES_PER_ENGINE 3
```

Comment:
I think it should be max(I915_SAMPLE_SEMA, extract_index_from(I915_PMU_RC6_RESIDENCY))

Reply:
Let's discuss it verbally.